Extension/popup.js

```diff
@@ -6,35 +6,43 @@ function onWindowLoad() {
     }
   });
 
+  function showKeybinding(command) {
+    if (command.shortcut) {
+      document.getElementById(command.name).innerHTML += ' (' + command.shortcut + ')';
+    }
+  }
+
+  var commands = chrome.commands.getAll(function(commands) {

```

Comment:
Ah. I had originally saved the commands away and foreached them (or whatever). Then I thought of using map, but forgot to remove the variable. Good catch.